ci/terraform/ticf-cri-stub/ticf-cri-stub-gateway.tf

```diff
@@ -0,0 +1,181 @@
+data "aws_vpc" "auth_shared_vpc" {
+  filter {
+    name   = "tag:Name"
+    values = ["${var.environment}-shared-vpc"]
+  }
+}
+
+data "aws_vpc_endpoint" "auth_api_vpc_endpoint" {
+  vpc_id       = data.aws_vpc.auth_shared_vpc.id
+  service_name = "com.amazonaws.eu-west-2.execute-api"
+  tags = {
+    environment = var.environment
+    terraform   = "core"
+  }
+}
+
+
+resource "aws_api_gateway_rest_api" "ticf_cri_stub" {
+  name = "${var.environment}-di-ticf-cri-stub"
+  body = jsonencode({
+    openapi = "3.0.1"
+    info = {
+      title = "${var.environment}-di-ticf-cri-stub"
+    }
+    paths = {
+      "/auth" = {
+        post = {
+          parameters = [{
```

Comment:
no parameters are needed (unlike for interventions), all the parameters will go in the body: https://github.com/govuk-one-login/fraud-ticf-cri/blob/main/api/criAuthGatewayApi.yaml